<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::ForceTabbedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background: transparent;
background-color: rgb(248, 248, 255);</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>1221</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    font: 14pt &quot;黑体&quot;;
    margin-top: 1ex;
    border-width:0.5px;  
    border-style:solid;
    border-color: rgb(51, 153, 255);  
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left:15px;      
    margin-left: 5px;
    margin-right: 5px;
    padding:1px;
}</string>
    </property>
    <property name="title">
     <string>L1</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>标准总占比</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>65</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>当前总占比</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>车型</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>1221</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    font: 14pt &quot;黑体&quot;;
    margin-top: 1ex;
    border-width:0.5px;  
    border-style:solid;
    border-color: rgb(51, 153, 255);  
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left:15px;      
    margin-left: 5px;
    margin-right: 5px;
    padding:1px;
}</string>
    </property>
    <property name="title">
     <string>L2</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>标准总占比</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>65</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>当前总占比</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>车型</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="but_set">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>63</width>
      <height>60</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icon/setting.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="but_query">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>710</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
   font: 14pt &quot;黑体&quot;;
   color: rgb(248, 248, 255);
   border-style:solid;
   background-color: rgb(51, 153, 255);
}
QPushButton:pressed{
	Qt::ToolButtonTextUnderIcon
}</string>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="but_close">
    <property name="geometry">
     <rect>
      <x>1329</x>
      <y>0</y>
      <width>38</width>
      <height>38</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icon/close.bmp);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="show_time">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>730</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color: rgb(51, 153, 255);
font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="wifi">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>735</y>
      <width>29</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/wifi.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
